### DISCUSSION TITLE
Adapting to different types of cultures

### DISCUSSION THREAD

**Discussion by ddocta** [tayo-k/bmd-v2#6] [General]
As a Product Manager at a few different companies, I've seen how widely the culture can range between tech companies, both on the technical and non-technical side. What are some ways that BMD can be altered to a specific company's culture and its particular nuances?

**Comment by tayo-k** [comment]
Great question, and it underscores the central thesis of BMD: while "Culture" _is_ unique between companies, [Cultural Outcomes](https://bmdhub.org/content/elements.html#outcomes-cultural-outcomes) are fairly consistent across a broad section of the industry. It's usually the same sort of "We encourage our developers to innovate and experiment", "Prioritize quality over speed" or even "Prioritize speed over quality". This way the uniqueness in "Culture" will come from:

1. The prioritizations of the Cultural Outcomes that make up the company's overall "Culture". This way, one company might prioritize speed of delivery over quality of code, or automated tests, or even developer happiness. Another might prioritize quality and security of code over everything else
2. The specific People, Processes and Tools (a.k.a [Components](https://bmdhub.org/content/elements.html#components)) that the company has at its disposal, to invest in the specific Outcomes they've prioritized. 

Putting those two cultural differentiators together, any given company can then 

1. Standardize its approach to delivering its desired Cultural Outcomes by making sure that the prioritizations of outcomes is transparently documented and communicated across the organization
2. Document the specifics of available and applicable people, processes and tools that they've decided to apply towards the documented Cultural Outcomes. They can also borrow from public libraries of Cultural Components and [Capabilties](https://bmdhub.org/content/elements.html#capabilities) (e.g. [Google's DevOps Library](https://cloud.google.com/architecture/devops)), to be able to reuse tried and tested ingredients, in delivering their desired Cultural Outcomes
3. Decide which [Signals](https://bmdhub.org/content/elements.html#signals) are of interest to them in monitoring their stated Cultural Outcomes. Because, while Outcomes may be broadly applicable, Signals can mean different things in different contexts. A company that operates a single-tenant on-prem/disconnected product might have longer [Mean Time to Resolution ](https://www.atlassian.com/incident-management/kpis/common-metrics) than a company that operates a Cloud-based multi-tenant SaaS product, therefore, MTTR might not be as useful a signal to the former company when measuring their DevOps culture. 

Hope this helps!
